src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -330,6 +356,11 @@ ApiGlobe.prototype.orbit = function(value) {
 ApiGlobe.prototype.setLayerOpacity = function(id, visible) {
 
     this.scene.getMap().setLayerOpacity(id, visible);
+    eventLayerChanged.layerId = id;
+    eventLayerChangedOpacity.layerId = id;
```

Comment:
Called when the opacity of a layer has changed, return an object with two properties, id and opacity